### PR TITLE
testing/backup-manager: new aport

### PR DESCRIPTION
**Pull request by uu**
Bash based backup manager from https://github.com/sukria/Backup-Manager.

### CODE REVIEW

File: testing/backup-manager/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+builddir="$srcdir/$_pkgreal-$pkgver"
+
+prepare() {
+	sed -e "/^PERL5DIR/s/sitelib/vendorlib/" \
+	    -e "/sed/s:=\$(DESTDIR)/:=:" \
+	    -i Makefile
+}
+
+package() {
+	make PREFIX=/usr DESTDIR="$pkgdir" -C "$builddir" install_binary
```
Comment (Cogitri) [inline testing/backup-manager/APKBUILD]: I don't think the `-C "$builddir"` is actually required here

Comment (uu) [reply testing/backup-manager/APKBUILD]: I'll make a new PR, you can close this one. Thanks;)

Comment (Cogitri) [reply testing/backup-manager/APKBUILD]: Ah, you can just force push here

Comment (uu) [reply testing/backup-manager/APKBUILD]: I removed that branch;) 
Here's new PR #11908

**Pull request closed without merging**